crates/miden-objects/src/account/file.rs

```diff
@@ -63,6 +63,7 @@ impl AccountFile {
 
 impl Serializable for AccountFile {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        target.write_bytes("acct".as_bytes());
```

Comment:
nit: I would probably define a constant for "acct" at the top of the file. Something like:
```rust
const MAGIC: &str = "acct".
```
And then use the same constant here and in `read_from()`.

We should probably do the same for the `NoteFile` as well.